pathd/path_pcep.c

```diff
@@ -248,31 +248,43 @@ DEFUN(show_pcep_counters, show_pcep_counters_cmd, "show pcep counters",
 }
 
 DEFUN_NOSH(pcep_cli_pcc, pcep_cli_pcc_cmd,
-	   "pcc [ip A.B.C.D] [port (1024-65535)] [force_stateless]",
+	   "pcc [ip A.B.C.D | ipv6 X:X::X:X] [port (1024-65535)] [force_stateless]",
```

Comment:
I think this should be `<ip A.B.C.D | ipv6 X:X::X:X>` otherwise you could specify both `ip` and `ipv6`.